cmake/pcl_targets.cmake

```
@@ -385,6 +385,21 @@ macro(PCL_ADD_TEST _name _exename)
   # must link explicitly against boost only on Windows
   target_link_libraries(${_exename} ${Boost_LIBRARIES})
 
+  #Only applies to MSVC
+  if(MSVC)    
+    #Requires CMAKE version 3.13.0
+    if(${CMAKE_VERSION} VERSION_LESS "3.13.0" AND (NOT ArgumentWarningShown))
```

CMake automatically evaluates variables, so there is no need for `${}`

```suggestion
    if(CMAKE_VERSION VERSION_LESS "3.13.0" AND (NOT ArgumentWarningShown))
```